README.md

```diff
@@ -1,3 +1,20 @@
+[dallin]("./src/images/dammad-logo.png")[mikayda] \*\*
+\*\*[dallin: './src/images/dallin.jpeg'][mikayda: './src/images/mikaydapic.jpeg']
+
+# DAMMAD.IO
+
+We made mad apps in damn good time.
```

Comment:
I like this - should change **damn** to **dam**. Also on LinkedIn we say "Making DAM good software with MAD design" I liked your version of this, but we should stay consistent, so let me know what you think.

Reply:
OO yes I forgot where we wrote that and was shooting from the hip - let me change it to the LinkedIn